src/DataBase/UserDatabase.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System.Linq;
+using ePiggyWeb.DataBase.Models;
+using ePiggyWeb.Utilities;
+
+namespace ePiggyWeb.DataBase
+{
+    public static class UserDatabase
+    {
+        public static int Register(string email, string pass)
+        {
```

Comment:
Since almost all changes in this request are moving code around, I wanted to ask did anything change in this file? Or is it just a copy? And where are the changes made to code?

Reply:
Because I renamed the file, none of the changes are being shown :(

Reply:
Well I did not spot any code standard violations, so I think everything is OK